java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1096,7 +1096,7 @@ public void ltrim_existing_non_existing_key_and_type_error(BaseClient client) {
         assertArrayEquals(new String[] {"value1", "value2"}, client.lrange(key, 0, -1).get());
 
         // `start` is greater than `end` so the key will be removed.
-        assertEquals(OK, client.ltrim(key, 4, 2).get());
```

Comment:
why did you remove the non `GlideString` test cases?
You should have added a `GlideString` in addition to `String`, not instead

Reply:
there is a call to `ltrim` with strings few lines above (line 1095)